### PR TITLE
Default structure

### PR DESCRIPTION
**Pull request by auxfuse**
# 🎉 🎉  Our first big PR 🤜 

## This PR includes:
* Default folder structure
* static folders and files for css/js/image
* app.py file with basic Flask functionality for testing the base.html inheritance to the index.html file
* gitignore file with as many ignorable things that I can think of right now
* Procfile used for deployment later
* requirements.txt file
* default templates folder with base.html parent defined
* base.html file is loading bootstrap 5 css&js links, and has default content block
* child pages templates folder, with default index.html and some boilerplate from bootstrap 5 docs to test the framework
css file contains a CSS reset, some root variable instances, Roboto & Arvo accessible text imports, and the root html font size set to 10px (<--- more on this later :smile: )
* js file contains a simple console log for now, and it is running fine on the render in the browser terminal.

This PR does not include any major breaking changes to the current structure of the project.

### Snapshots

* File directory setup:
image

* Default test render of the Flask application in the browser
    * this snapshot details the use of the Font imports, CSS rendering & Bootstrap rendering also.
image

## Testing

All testing so far has been done on the default file structure. No other testing necessary.
List of tests done:
* CSS file renders and displays correctly ------------ ✔️ 
* JS file renders and displays default log ----------- ✔️ 
* Flask app runs in browser ------------------------- ✔️ 
* Bootstrap render as expected via CDN links ----- ✔️

### CODE REVIEW

**Review by StephenJ2020 — COMMENTED** [review]
Quick question, those "Load diff" files that were deleted, am I correct in thinking they are files that were automatically included with the CI Template but that aren't required for this App?

The Procfile looks good, the extra line of empty space has been removed to avoid any issue.

For the app.py file on line 16 do we need to make that an"int"? See comment for example pls. We'll also need to add a blank line at the end I think, but that's no biggie!

requirements.tkt looks fine for now but I guess we may need to add to this as we progress with the build.

style.css looks fine

main.js looks fine

The base.html template looks grand but we will need to add in some additional metadata for SEO

index.html looks fine

File: app.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+@app.route("/")
+def index():
+    return render_template('pages/index.html')
+
+
+if __name__ == "__main__":
+    app.run(
+        host = os.environ.get('IP', '127.0.0.1'),
+        port = os.environ.get('PORT', '5000'),
```
Comment (StephenJ2020) [inline app.py:16]: Should this line be an int? : port=int(os.environ.get("PORT")),
Or is that only if we don't specify the actual PORT number within the parentheses?

File: templates/base.html [hunk 2]
```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
```
Comment (StephenJ2020) [inline templates/base.html:5]: <meta name="description" content="">
    <meta name="keywords" content="">
    <meta name="author" content="">

File: app.py (hunk 1 above)
Comment (Joan-Amudu) [reply app.py:16]: @StephenJ2020 I don't think we need to specify data types when setting environment variables.

**Review by StephenJ2020 — APPROVED** [review]
All comments discussed and agreed by team

**Pull request merged**